plugins/builder/osbuild.py

```diff
@@ -379,6 +394,10 @@ def main():
                         action="append", type=str, default=[])
     subpar.add_argument("--koji", metavar="URL", help='The koji url',
                         default=DEFAULT_KOJIHUB_URL)
+    subpar.add_argument("--cert", metavar="cert", help='The client SSL certificates to use',
```

Comment:
Do we also need/want to support the key being separate from the cert? In the same way for instance `curl` or `koji` does?

Reply:
Already supported, you pass `--cert` multiple times.